src/bindings/js/node/lib/addon.ts

```diff
@@ -0,0 +1,125 @@
+interface Core {
+  compileModel(): CompiledModel;
+  readModelAsync(): Promise<CompiledModel>;
+  readModel(modelPath: string, binPath?: string): Model;
+}
+interface CoreConstructor {
+  new(): Core;
+}
+
+interface Model {
```

Comment:
Model has an empty, obligatory constructor

Reply:
Extra constructors, the result of which is not useful from JS side are removed from types definition